### DISCUSSION TITLE
magpie为什么不能缩放vmware workstation?

### DISCUSSION THREAD

**Discussion by 300206** [Blinue/Magpie#940] [General]
vmware workstation的自带的缩放算法非常烂，老游戏画面很差，我使用vm的 unity mode 开magpie缩放，能看到画面放大，但不能做任何操作，鼠标也没有。

**Comment by hooke007** [comment]
dup https://github.com/Blinue/Magpie/issues/921

**Reply by Blinue** [reply]
可能不一样，unity模式将虚拟机中的窗口映射到宿主机上，所以缩放时使用的是宿主机的显卡。但我这里不知为何用不了 unity 模式

**Reply by 300206** [reply]
> 可能不一样，unity模式将虚拟机中的窗口映射到宿主机上，所以缩放时使用的是宿主机的显卡。但我这里不知为何用不了 unity 模式

magie是在host system中启动的，unity模式中启动游戏画面如下，一个小分辨率窗口位于主系统窗口的的左上，调用magie热键可以对这个小窗口放大，但放大后什么也做不了,就是一个静止画面
![Uploading 图片.png…]()

**Reply by 300206** [reply]
我用的系统是 win10 2019 lstc ，magpie只能用0.9.1。换了win10 2021 ltsc和magpie 0.11后，可以正常缩放unity模式了，完美。

**Comment by Blinue** [comment]
我这里 unity 模式的窗口也可以正常缩放。你的情况是所有窗口都有问题还是只有游戏有问题？

![image](https://github.com/Blinue/Magpie/assets/34770031/333b05a7-40ae-469c-8ab6-a572979ad32e)

**Reply by Blinue** [reply]
VMware 版本：17.5.0
宿主机 OS：Win10 22H2
虚拟机 OS：Win10 22H2
